_sass/ecommerce.scss

```diff
@@ -154,17 +151,21 @@
   }
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 700px) {
```

Comment:
Why do we need this media?

Reply:
@katatsu12 We need this media because of cards in this page are big and need more width. With satndart styles they look disproportionate, because then the height is high